### ISSUE TITLE
LICENSE? 

### ISSUE THREAD

**Issue by Levi-Armstrong** [jrgnicho/geometric_support_tools#1]
What is the license of the node written in this package?

**Comment by gavanderhoorn** [comment]
Looks like Apache 2?

https://github.com/jrgnicho/ros_support_tools/blob/9cf725265c267e763c359bfa46472d016f260a15/convex_decomposition/package.xml#L7

(just happened to come across this pkg, not 'spying' on Jorge :))

**Comment by Levi-Armstrong** [comment]
Yea I was not sure if that applied if the file did not have it at the top of the file. Usually every file has the license at the top so I was not sure.

**Comment by jrgnicho** [comment]
I didn't specified a license on the node source file so we may be able to add one of our choosing and update the package xml

**Comment by Levi-Armstrong** [comment]
I was wanting to use parts of it in one of the tools I am developing.

**Comment by gavanderhoorn** [comment]
It seems like V-HACD is being wrapped and the license for that is [BSD 3-Clause](https://github.com/kmammou/v-hacd/blob/05823000bfd5cc0ca086014f6129cc7a5e94e23c/LICENSE).

**Comment by jrgnicho** [comment]
@gavanderhoorn does that mean that the license of the V-HACD code shall be propagated to the node code as well?

**Comment by gavanderhoorn** [comment]
No, but you must acknowledge the license afaik (just add a second `license` tag to the manifest).

See [For the purposes of being included in an Apache product, which licenses are considered to be similar in terms to the Apache License 2.0?](http://www.apache.org/legal/resolved.html#category-a) for more info.

**Comment by jrgnicho** [comment]
@gavanderhoorn I see, thanks.  @Levi-Armstrong Is the licensing in the V-HACD code keeping you from using it?
